typeDefs.js

```diff
@@ -0,0 +1,77 @@
+const { gql } = require('apollo-server');
+const category = require('./modules/categories/categories.model');
+
+const typeDefs = gql`
+  type Language {
+    lang: String!
+    value: String!
+  }
+
+  type ImageSet {
+    large: String
+    medium: String
+    small: String
+    thumbnail: String
+  }
+
+  type PrimaryImage {
+    primary: ImageSet!
+    additional: [ImageSet!]
+  }
+
+  type ConvertOption {
+    name: String!
+    exchangeRate: Float!
+  }
+
+  type Currency {
+    _id: ID!
+    date: String!
+    convertOptions: [ConvertOption!]
+  }
+
+  type Subcategory {
+    _id: ID!
+    categoryCode: String!
+    name: [Language!]
+    images: ImageSet
+    available: Boolean!
+  }
+
+  # type Category {
+  #   _id: ID!
+  #   categoryCode: String!
+  #   name: [Language!]
+  #   images: ImageSet
+  #   subcategories: [Subcategory!]
+  #   available: Boolean!
+  # }
```

Comment:
remove commented code. Comments should be only for information about what does code do or why you wrote it

Reply:
fixed